src/reducers/global.ts

```diff
@@ -340,13 +360,22 @@ export const global = (state = defaultState, action: any) => {
         tree,
         treeTitle,
         divisionOptions,
+        showTreeFormatError: false,
         mrcaOptions: traverse_preorder(tree).filter(
           (node: Node) => node.children.length >= 2
         ),
         cladeSliderField,
         cladeSliderValue: formatMrcaSliderOptionValue(tree, cladeSliderField),
         mrca: tree,
         metadataCensus: { ...state.metadataCensus, ...treeMetadata },
+        fetchData: {
```

Comment:
The code as written will work, but it's probably better practice to reuse the initial default to overwrite. That way if the internal structure for `fetchData` changes down the line, this portion of the code doesn't need to know any of those implementation details. Instead, here you'd do this:
```
fetchData: {
  ...defaultState.fetchData,
},
```
This ensures it goes back to whatever the default state winds up being down the road and it's clearer to read if you aren't familiar with the `fetchData` structure since the purpose is just to reset to default.

Reply:
Oooh good call, thanks!